### PR TITLE
drivers: led: Fix uninitialized variable

### PR DESCRIPTION
**Pull request by rruuaanng**
Fix uninitialized ret variables scanned by Coverity.

from:
https://github.com/zephyrproject-rtos/zephyr/issues/81958

### CODE REVIEW

**Comment by simonguinot** [comment]
Hi @rruuaanng. Thanks for taking care of this issue. Please see my comment above.

File: drivers/led/is31fl3194.c [hunk 1]
```diff
@@ -80,7 +80,7 @@ static int is31fl3194_set_color(const struct device *dev, uint32_t led, uint8_t
 {
 	const struct is31fl3194_config *config = dev->config;
 	const struct led_info *info = is31fl3194_led_to_info(config, led);
-	int ret;
+	int ret = 0;
```
[the remark this answers is not in the document]
Comment (rruuaanng) [reply drivers/led/is31fl3194.c:83]: Sorry to have kept you waiting, I think your suggestion is very good. After `ret is initialized to 0`, we should `remove the if`, I will fix it in two commits :)

[the remark this answers is not in the document]
Comment (simonguinot) [reply drivers/led/is31fl3194.c:83]: Sorry I don't see what is wrong this the code example I proposed. The last call to `i2c_reg_write_byte_dt` is only executed if the color registers are updated.

**Comment by jilaypandya** [comment]
Hi @rruuaanng, is it okay if we close this PR. Since #82679 is already merged, let's see if sca is happy with the solution or not :)

**Comment by rruuaanng** [comment]
Thank you for your reminder. I have closed it.

**Comment by rruuaanng** [comment]
I agree with @simonguinot suggestion, but it is likely to be regarded as an appearance change. I think we can change it in the future. That is, when there are other problems with the driver. So, thank you :--)

**Pull request closed without merging**